### PR TITLE
First ionic test review

### PR DESCRIPTION
**Pull request by lmveloso**
- added some suggested test scenarios
- resetting mock calls after each inside
- changed the way `capacitor.config.ts` is being loaded for verification test.
- left some messages here and there with suggestions
- Exposing permission status inside `tab3.page.html` for user feedback so applicant can focus on the main goals
- Is the applicant supposed to fix the `ImagePicker` import inside `tab3.page.ts`?
- Is the applicant supposed to fix the `Device` mock inside `__mocks__/@capacitor` ?
- Is the applicant supposed to properly create a spy object for `ImagePicker` inside `tab3.page.spec.ts`? Suppose so, right?
- added a brief `README.md`
- fixed few typos

### CODE REVIEW

File: src/__mocks__/@capacitor/device.ts [hunk 1]
```diff
@@ -11,5 +11,6 @@ export type OperatingSystem = _OperatingSystem;
 
 export const Device = {
   getId: (): Promise<DeviceId> => Promise.resolve(void 0),
-  // getInfo: (): Promise<DeviceInfo> => Promise.resolve(void 0),
+  // LV REVIEW: QUESTION? Is the applicant supposed to fix the tests by touching this?
```
Comment (matheuscas) [inline src/__mocks__/@capacitor/device.ts:14]: If so, I'd delete it.

File: src/app/tab2/tab2.page.ts [hunk 2]
```diff
@@ -26,9 +26,18 @@ export class Tab2Page {
     this.showPermissionError = false;
     this.positionError = undefined;
 
-    const permissionStatus = await Geolocation.requestPermissions({
-      permissions: ['location'],
-    });
+    try {
```
Comment (matheuscas) [inline src/app/tab2/tab2.page.ts:29]: Shouldn't this be the applicant's job?

File: src/app/tab2/tab2.page.ts [hunk 3]
```diff
@@ -49,7 +58,12 @@ export class Tab2Page {
       {
         enableHighAccuracy: true,
       },
-      (position, error) => {}
+      (position, error) => {
```
Comment (matheuscas) [inline src/app/tab2/tab2.page.ts:61]: Also this.

File: src/app/tab3/tab3.page.ts [hunk 4]
```diff
@@ -9,24 +10,46 @@ import { ImagePicker } from '@awesome-cordova-plugins/image-picker';
...
   // TODO: if user has not the permission request permission, do not get pictures automatically
   // TODO: Request maximum 5 pictures
 
-  async ionViewWillEnter() {}
+  async ionViewWillEnter() {
+  }
 
   async ionViewDidEnter() {
     this.error = undefined;
+    try {
```
Comment (matheuscas) [inline src/app/tab3/tab3.page.ts:29]: Same comment from above.

File: src/app/tab2/tab2.page.ts (hunk 2 above)
Comment (lmveloso) [reply src/app/tab2/tab2.page.ts:29]: Yes, it is here just for reference and should not be part of the code delivered to applicants.

**Pull request merged**